src/services/blockchain/__tests__/eth-bc-service.test.ts

```diff
@@ -93,35 +91,27 @@ describe('ETH service connected to ganache', () => {
 
     test('gas price increase math', () => {
       const gasEstimate = {
-        maxFeePerGas: BigNumber.from(2000),
-        maxPriorityFeePerGas: BigNumber.from(1000),
-        gasPrice: BigNumber.from(0),
+        maxFeePerGas: BigInt(2000),
```

Comment:
```suggestion
        maxFeePerGas: 2000n,
```

`123123n` is a bigint literal in JS.